xml/installation-installation-configure-deployer-repos.xml

```diff
@@ -0,0 +1,499 @@
+   server via <literal>NFS</literal>, or to loop mount a copy of the
+   installation images.
+   </para>
+   <para>
+    The following product media must be copied to the specified
+    directories:
+   </para>
+   <table>
+    <title>Local Product Repositories for &cloud;</title>
+    <tgroup cols="2">
+     <colspec colnum="1" colname="1" colwidth="40*"/>
+     <colspec colnum="2" colname="2" colwidth="60*"/>
+     <thead>
+      <row>
+       <entry>
+        <para>
+         Repository
+        </para>
+       </entry>
+       <entry>
+        <para>
+         Directory
+        </para>
+       </entry>
+      </row>
+     </thead>
+     <tbody>
+      <row>
+       <entry>
+        <para>
+         &cloudos; DVD #1
+        </para>
+       </entry>
+       <entry>
+        <para>
+         <filename>/srv/www/suse-12.3/x86_64/repos/SLES12-SP3-Pool</filename>
+        </para>
+       </entry>
+      </row>
+      <row>
+       <entry>
+        <para>
+         &productname; &productnumber; DVD #1
+        </para>
+       </entry>
+       <entry>
+        <para>
+         <filename>/srv/www/suse-12.3/x86_64/repos/Cloud</filename>
+        </para>
+       </entry>
+      </row>
+     </tbody>
+    </tgroup>
+   </table>
+   <para>
+    The data can be copied by a variety of methods:
+   </para>
+   <variablelist>
+    <!-- sec.depl.adm_conf.repos.product.media -->
+    <varlistentry>
+     <term>Copying from the Installation Media</term>
+     <listitem>
+     <para>
+      We recommended using <command>rsync</command> for copying. If the
+      installation data is located on a removable device, make sure to mount
+      it first (for example, after inserting the DVD1 in the &admserv; and
+      waiting for the device to become ready):
+     </para>
+     <bridgehead renderas="sect4">&cloudos; DVD#1</bridgehead>
+     <screen>mkdir -p &www_dir;/repos/SLES12-SP3-Pool
+mount /dev/dvd /mnt
+rsync -avP /mnt/ &www_dir;/SLES12-SP3-Pool/
+umount /mnt</screen>
+     <bridgehead renderas="sect4">
+      &productname; &productnumber; DVD#1
+     </bridgehead>
+     <screen>mkdir -p &www_dir;/repos/Cloud
+mount /dev/dvd /mnt
+rsync -avP /mnt/ &www_dir;/repos/Cloud/
+umount /mnt</screen>
+     </listitem>
+    </varlistentry>
+    <varlistentry>
+     <!-- sec.depl.adm_conf.repos.product.remote -->
+     <term>Copying from a Remote Host</term>
+     <listitem>
+      <para>
+       If the data is provided by a remote machine, log in to that machine and
+       push the data to the &admserv; (which has the IP address <systemitem
+       class="etheraddress">192.168.124.10</systemitem> in the following
+       example):
+      </para>
+      <bridgehead renderas="sect4">&cloudos; DVD#1</bridgehead>
+      <screen>mkdir -p &www_dir;/install
+rsync -avPz <replaceable>/data/SLES-12-SP3/DVD1/</replaceable> <replaceable>192.168.124.10</replaceable>:&www_dir;/repos/SLES12-SP3-Pool/</screen>
```

Comment:
Not entirely sure but I think ardana is using different default addresses for the equivalent of the admin network. We might want to stick with those.

Reply:
Sure, will just have to figure out what that is (something like 192.168.245.? ?)

Reply:
I chose 192.245.10